### PR TITLE
OC-7568

### PR DESCRIPTION
**Pull request by hieudd**
Fix /ListStudyUser does not show removed user role

### CODE REVIEW

**Comment by sdibona** [comment]
Hello,

Thank you for this bug fix submission.  The logic changes look good.  
I am still reviewing the '*.properties' file changes and will provide a further update on Monday morning.

Once the code review is completed, would you please re-create this pull request into the newly created OpenClinica:3.13 branch? We'd like to include it in the next 3.x release.

It's currently open against OpenClinica:master and contains many additional changes for 4.x which we don't want pulled into 3.x. To get around this you will need to create a branch off of 3.13. Then execute 'git cherry-pick' to apply only this commit to your 3.13 branch. Then re-create the pull request. If this is a problem, please let me know and I'll assist with getting these changes into the 3.13 branch.

Thank you,
Steve DiBona

**Comment by hieudd** [comment]
Sure, please tell me when you ready. I will re-create this full request into the newly created OpenClinica:3.13 branch. Will contact you if need support.

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 1]
```diff
@@ -924,20 +924,6 @@ you_are_not_owner_some_items_cannot_delete = You are not the owner of some items
...
-you_can_access_odm_13 = You can access your ODM 1.3 XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_odm_13_xml = You can access your ODM 1.3 w/OpenClinica Extension XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_spss = You can access your SPSS files <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_tab_delimited = You can access your tab-delimited file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_review_the_data = <p>You can review the entered data <a href='{0}ListStudySubjects'>here</a>.
-
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: All of the strings defined here are still referenced by ExampleSpringJob.java and should not be removed.

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 2]
```diff
@@ -996,6 +982,8 @@ your_listed_crf_in_the_file = Your listed Event CRF in the file {0} does not exi
 
 your_message_sent_succesfully = Your message has been sent successfully.
 
+your_message_sent_succesfully_to = Your message has been sent successfully {0}.
+
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: This string does not appear to be referenced in the code base.  Should be removed.

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 3]
```diff
@@ -1037,11 +1024,7 @@ lockout_changes_have_been_saved=The changes you made to the Lockout Configuratio
 job_ran_for=Job ran for
 job_ran_but_no_files=Job ran but no real files found.
 job_failure_for=Job FAILURE for
-your_job_ran_success_html=,<br/>Your job ran successfully with the following messages returned.
 please_review_the_data_html=Review the data, or modify the files accordingly so that they can be imported.<br/>
-unreadable_file=Unreadable File
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: This line is referenced in ImportSpringJob.java.  However the reference is in dead code in the class.  For consistency, either the string should be left here or the dead code in ImportSpringJob.java should also be removed.

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 4]
```diff
@@ -1077,15 +1060,11 @@ right_item_length_error=Right item text cannot be more than 4000 characters
 item_header_length_error=Item header cannot be more than 2000 characters
 item_subheader_length_error=Item subheader cannot be more than 240 characters
 regexp_errror_length_error=Regexp error cannot be more than 255 characters
-group_label_length_error=Group Label cannot be more than 255 characters
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: This string is still referenced in SpreadSheetTableRepeating.java.  Should not be deleted.

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties [hunk 5]
```diff
@@ -1077,15 +1060,11 @@ right_item_length_error=Right item text cannot be more than 4000 characters
...
 radio_with_default = OpenClinica does not support DEFAULT_VALUES with radio RESPONSE_TYPES. Change the RESPONSE_TYPE for this item
 change_radio = , or remove the value in the DEFAULT_VALUE column.
-group_label_length_error=Group Label cannot be more than 255 character
 repeating_group_error = Repeating Group must be true/false.
 repeat_number_none_repeating = Repeat Number must be blank for non-repeating groups.
-repeat_max_none_repeating = Repeat Max must be blank for non-repeating groups.=======
 group_header_length_error=Group Header cannot be more than 255 characters=======
-repeat_max_none_repeating = Repeat Max must be blank for non-repeating groups.
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: Still referenced in SpreadSheetTableRepeating.java.  Should not be deleted.

File: web/src/main/resources/org/akaza/openclinica/i18n/words.properties [hunk 6]
```diff
@@ -3094,7 +3096,3 @@ preview=Preview
 choose_crf_migration_batch_instruction_key=Please choose the current CRF Version, New CRF Version, applicable Sites, and applicable Events for batch CRF version migration. This action will be audited and appear in each Subject's audit log.
 migrate=Migrate
 
-repeating_layout_group_item_assigned_to_wrong_group=All items located directly or indirectly in a repeating layout group must be assigned to the same data group.
-nested_repeat_group_not_allowed=Repeating layout groups can be included at most one time in a nested layout groups structure
-duplicate_item_name=Item Names are unique per CRF
-group_name_missing_for_this_item=CRFs must have a data group defined for every Item
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/words.properties]: This group of strings was defined in a 4.x commit and will not exist when the pull request is re-created against 3.13

File: web/src/main/resources/org/akaza/openclinica/i18n/words.properties [hunk 7]
```diff
@@ -1593,6 +1593,8 @@ remove_filter = Remove Filter
 
 remove_previous_same_version_error = Remove Previous Same Version Error
 
+role_status = Role Status
```
Comment (sdibona) [inline web/src/main/resources/org/akaza/openclinica/i18n/words.properties]: New strings should be defined at the bottom of the file to make it easier for people updating their translations to find them.

**Comment by hieudd** [comment]
Thanks for your response. We will fix and update in the next week. 

Thank you very much!

File: web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties (hunk 1 above)
Comment (sinzii) [inline web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties]: - I found a duplicated string for you_can_access_tab_delimited (line 937 and 1050) and for you_can_review_the_data (line 339 and 1046), that why i remove them. Please check this again.

**Pull request closed without merging**